README.md

```diff
@@ -138,10 +138,9 @@ Caveats:
    - start from a baseline configuration with no indexing plugins installed.
 
 Prerequisites:
-   - Install a Java 8 JRE or JDK (OpenJDK versions will suffice).
    - Download, untar, and run the ElasticSearch package or equivalent:
```

Comment:
Is this not necessary because all of our supported platforms have the right JRE / JDK? Or, are we assuming that "Download, untar, and run the ElasticSearch package" includes following directions in which the user is instructed on which version(s) to use?

Reply:
Elasticsearch includes the appropriate JRE/JDK so we don't need to handle java at all.